package.json

```diff
@@ -4,7 +4,9 @@
   "private": true,
   "homepage": "http://aneagoie.github.io/fullstacktrends",
   "dependencies": {
+    "chart.js": "^2.7.3",
```

Comment:
do we need this or just `react-chartjs-2`?

Reply:
![chartjs](https://user-images.githubusercontent.com/38914008/50185729-b1f52700-033e-11e9-8cce-a7cbb6c2ad51.png)